### DISCUSSION TITLE
How can i read tonic solfa/solfege using Audiveris

### DISCUSSION THREAD

**Discussion by vhmangaiha** [Audiveris/audiveris#667] [Q&A]
I am trying to create an application that can read tonic solfa/solfege in  a pdf/image and transform it into Staff Notation and output a midi file.
Is it possible to  read tonic solfa/solfege  using Audiveris. If so what is the way to approach this problem.

**Comment by hbitteur** [comment]
Sorry, I never heard of tonic sol-fa.
Could you post an example of a related input image?

**Comment by vhmangaiha** [comment]
[https://www.singsolfa.com/about/methodology/](url)
[https://alchetron.com/Tonic-sol-fa](url)
This is an example of tonic solfa notation

**Comment by hbitteur** [comment]
@vhmangaiha 
Following your url, here is a sol-fa  example:

![image](https://github.com/Audiveris/audiveris/assets/22053634/673b2df4-10b5-439e-82a9-823c020205c3)

This is not at all the common music notation (staff lines, note heads, stems, etc) that Audiveris can process.
Sorry.

**Comment by vhmangaiha** [comment]
Will it be possible to read the above using Audiveris , if changes are made to the code?

**Comment by hbitteur** [comment]
I think not. Changes would be so heavy that we'd better start a new project from scratch!
